rna/management/commands/rnasync.py

```diff
@@ -23,10 +31,11 @@ def model_params(self, models):
 
     def handle(self, *args, **options):
         rc = clients.RNAModelClient()
-        model_params = self.model_params(rc.model_map.values())
+        model_params = self.model_params(rc.model_map.values(),
+                                         force=options['force'])
```

Comment:
won't this be `options['delete']` since you set `dest='delete'` in the `make_option` call?